packages/web-ui/src/Alert/Alert.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+import { COLOR_SCHEME, PropsWithSx } from '../types';
+import { AlertProps } from './Alert.props';
+import { COLORSCHEME_SELECTORS, DATA_ATTRIBUTES, px, withGlobalPrefix } from '../utils';
+import clsx from 'clsx';
+import { styled } from '../theme';
+import { colors } from '@utilitywarehouse/colour-system';
+import {
+  WarningMediumContainedIcon,
+  InformationMediumContainedIcon,
+  TickMediumContainedIcon,
+  CloseMediumIcon,
+  CloseSmallIcon,
+  ChevronRightMediumIcon,
+} from '@utilitywarehouse/react-icons';
+import { AlertTitle } from './AlertTitle';
+import { AlertText } from './AlertText';
+import { AlertLink } from './AlertLink';
+import { Flex } from '../Flex';
+import { UnstyledButton } from '../UnstyledButton';
+
+const componentName = 'Alert';
+const componentClassName = withGlobalPrefix(componentName);
+
+const StyledElement = styled('div')({
+  display: 'flex',
+  gap: px(8),
+  borderRadius: px(8),
+  borderWidth: px(1),
+  borderStyle: 'solid',
+  borderColor: 'var(--alert-border-color)',
+  flexDirection: 'row',
+  padding: px(16),
+  backgroundColor: 'var(--alert-background-color)',
+  color: 'var(--alert-text-color)',
+  [`:where([${DATA_ATTRIBUTES.orientation}="vertical"])`]: {
+    '> button[data-dismiss]': {
+      alignSelf: 'flex-start',
+    },
+  },
+  '> :where(svg, [data-icon])': {
+    color: 'var(--alert-icon-color)',
+  },
+  '> :where(button, a)': {
+    color: 'var(--alert-icon-color)',
+    alignSelf: 'center',
+    '&:hover': {
```

Comment:
```suggestion
     '@media (hover: hover)': {
      ':where(:hover)': {
```

Reply:
fyi: https://jackdomleo.dev/blog/2020/hover-css-media-query

Reply:
I mean that's a step too far, I'll have read though 😂

Reply:
In the example that's a neat way to show states by default for non-hover devices. Do we really need that here? I'm all for consistency but this seems overkill?

Reply:
sure, probably is 👍🏻